client/src/components/UploadProject.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from "react";
+import { useState } from "react";
 import { Button } from "./Button";
```

Comment:
Components should have a default export:

Look at this: https://github.com/DiscoDevs/DispoDisco/pull/76